cookbooks/fb_networkmanager/attributes/default.rb

```diff
@@ -0,0 +1,43 @@
+#
+# Cookbook:: fb_networkmanager
+# Recipe:: default
+#
+# Copyright (c) 2019-present, Vicarious, Inc.
+# Copyright (c) 2019-present, Facebook, Inc.
```

Comment:
Put 2020 in the fb byline

Reply:
This just gives you MORE copyright. :) But yeah will do.